words/teams.txt

```diff
@@ -1,27 +1,130 @@
+aerie
 alliance
 assembly
+bale
 band
+barrel
 batch
+bed
+bevy
+board
+brood
+building
 bunch
+business
+cackle
+camp
+cast
+catch
+cauldron
+charm
+chattering
+chime
+choir
+circle
+clan
+class
+clattering
+cloud
+clowder
 club
+cluster
+coalition
+colony
 combination
+committee
+company
 conglomerate
+congregation
+congress
+conspiracy
+convocation
+corporation
 coven
 crew
+culture
+dazzle
+descent
+doctrine
+drift
+drove
+exaltation
 faction
+faculty
+family
+flight
+fling
+flock
+flush
+gaggle
+galaxy
+game
 gathering
+gobble
 group
+gulp
+herd
+hive
+intrigue
+jury
+kettle
+kit
+knot
+labor
+lamentation
 league
+lease
 lineup
+litter
+murmuration
+mustering
+nest
+orchestra
 order
 organization
+ostentation
 outfit
+pace
 pack
+pandemonium
+parade
+parliament
 party
+phalanx
+piteousness
+pod
 posse
+prickle
+pride
+quiver
+raffle
+romp
+rookery
+sawt
+school
+scoop
+scream
+scury
+sedge
+sentence
+shadow
+shiver
+shrewdness
+sleuth
+sloth
 squad
+staff
```

Comment:
Like `support` maybe we should avoid this term?

Reply:
since this is teams, I think its ok?

Reply:
yeah probably!